src/tracer.test.js

```diff
@@ -482,6 +482,23 @@ describe('tracer', () => {
     expect(Http.hookHttp).toHaveBeenCalledTimes(1);
   });
 
+  test('trace; async local lambda no aws _X_AMZN_TRACE_ID', async () => {
+    delete process.env['_X_AMZN_TRACE_ID'];
```

Comment:
This line is not mandatory (because of the next line), right?

Reply:
No i want see that its not crushing when its missing

Reply:
Maybe we can do `parametrize`?  One with "_X_AMZN_TRACE_ID" and one without "_X_AMZN_TRACE_ID"?